Rules/Strings.resx

```diff
@@ -897,6 +897,18 @@
   <data name="AvoidTrailingWhitespaceError" xml:space="preserve">
     <value>Line has trailing whitespace</value>
   </data>
+  <data name="AvoidLongLinesName" xml:space="preserve">
+    <value>AvoidLongLines</value>
+  </data>
+  <data name="AvoidLongLinesCommonName" xml:space="preserve">
+    <value>Avoid long lines</value>
+  </data>
+  <data name="AvoidLongLinesDescription" xml:space="preserve">
+    <value>Each line should be not be too long.</value>
+  </data>
+  <data name="AvoidLongLinesError" xml:space="preserve">
+    <value>Line is too long</value>
```

Comment:
```suggestion
    <value>Line exceeds configured maximum length of {0} characters</value>
```

Reply:
Changed. Also updated the `DiagnosticRecord` construction to plug `MaximumLineLength` in.